### ISSUE TITLE
[CDCSDK] Tablet not found for a higher rf and number of masters

### ISSUE THREAD

**Issue by isha-yb** [yugabyte/yugabyte-db#12613]

**Comment by isha-yb** [comment]
For replication factor = 5 and number of masters = 3, a response error is returned while setting cdc checkpoint
```
resp error {
12187   status {
12188     code: NOT_FOUND
12189     message: "Tablet not found: 057e931d0f1047ef8d32e80ca5466c97"
12190     source_file: "../../src/yb/tserver/ts_tablet_manager.cc"
12191     source_line: 1698
12192     errors: "\020\002\000\000\000\000"
12193   }
12194 }
```

**Comment by sureshdash2022-yb** [comment]
We have fixed the issue in master and backported to 2.12 and 2.14
